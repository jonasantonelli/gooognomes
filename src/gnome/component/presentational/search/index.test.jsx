/**
 * @jest-environment jsdom
 */
import React from 'react';
import { shallow } from 'enzyme';
import Search from './index.jsx';

describe('Search Component', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Search />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should contain title class', () => {
    expect(wrapper.exists('.search')).toEqual(true);
  });

  it('should contain children example', () => {
    expect(wrapper.exists('input')).toEqual(true);
  });
});
