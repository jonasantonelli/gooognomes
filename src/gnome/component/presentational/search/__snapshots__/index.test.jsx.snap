// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Component should match snapshot 1`] = `
<div
  className="search"
>
  <input
    autoComplete="off"
    name="search"
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="Gnomes..."
    type="text"
  />
  <button
    onClick={[Function]}
    type="button"
  >
    Clean
  </button>
</div>
`;
